ors-api/src/test/java/org/heigit/ors/api/servlet/listeners/ORSInitContextListenerTest.java

```diff
@@ -16,16 +17,24 @@ void setUp(){
     }
     @ParameterizedTest
     @CsvSource(value = {
-            "https://example.com/test-repo/planet,      https://example.com, test-repo, planet",
-            "https://example.com/test-repo/planet/,     https://example.com, test-repo, planet",
-            "' https://example.com/test-repo/planet ',  https://example.com, test-repo, planet",
-            "' https://example.com/test-repo/planet/ ', https://example.com, test-repo, planet",
-            "' http://example.com/test-repo/planet/ ',  http://example.com, test-repo, planet"
+            "https://example.com/foo1/foo2/foo3/test-repo,      https://example.com, test-repo, true",
```

Comment:
first line here has the wrong expectation, expected url is https://example.com/foo1/foo2/foo3
```suggestion
            "https://example.com/foo1/foo2/foo3/test-repo,      "https://example.com/foo1/foo2/foo3, test-repo, true",
```